.github/workflows/pr-build-and-e2e-tests.yml

```diff
@@ -1,14 +1,14 @@
 name: Run tests against PR
 on:
-  pull_request:
-  workflow_dispatch:
+    pull_request:
+    workflow_dispatch:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
 
 env:
-  E2E_PLAYWRIGHT: ${{ false }}
+    E2E_PLAYWRIGHT: ${{ false }}
```

Comment:
AFAIK we're using 4 spaces in the default style guide for YAML files, so I think leaving these changes in spacing is OK?

Reply:
> we're using 4 spaces in the default style guide for YAML files

Good question, I am not sure if we're using a set style, but the file is still valid YAML and works as expected, so I think this should be OK as well.